### PR TITLE
Scripts to open Finder directory in Terminal and vice versa

### PR DESCRIPTION
**Pull request by japanese-goblinn**

### CODE REVIEW

File: navigation/open-finder-from-terminal.applescript [hunk 1]
```diff
@@ -0,0 +1,31 @@
+#!/usr/bin/osascript
+
+# Required parameters:
+# @raycast.schemaVersion 1
+# @raycast.title Terminal -> Finder
```
Comment (shpakovski) [inline navigation/open-finder-from-terminal.applescript:5]: Could you please rename to smth like “Open current iTerm directory in Finder”?

File: navigation/open-terminal-from-finder.applescript [hunk 2]
```diff
@@ -0,0 +1,52 @@
+#!/usr/bin/osascript
+
+# Required parameters:
+# @raycast.schemaVersion 1
+# @raycast.title Finder -> Terminal
```
Comment (shpakovski) [inline navigation/open-terminal-from-finder.applescript:5]: Same here, but vice versa 🙏

**Review by shpakovski — CHANGES_REQUESTED** [review]
Thanks!

File: navigation/open-finder-from-terminal.applescript (hunk 1 above)
Comment (japanese-goblinn) [reply navigation/open-finder-from-terminal.applescript:5]: yep

**Pull request merged**